pkg/resource/factory.go

```diff
@@ -93,3 +83,31 @@ func (rf *Factory) SliceFromBytes(in []byte) ([]*Resource, error) {
 	}
 	return result, nil
 }
+
+func (rf *Factory) Set(fs fs.FileSystem, ldr ifc.Loader, wd string) {
```

Comment:
Do we have to send wd?  Is this different from ldr.Root for the given use cases?

Regardless, needs comment explaining wd and why its different from ldr.Root

Reply:
Adding a `wd` is easy for this change, especially in the test files. `wd` is actually the same as `ldr.Root`. I'll remove this argument.